test/netcard_test.rb

```diff
@@ -69,9 +69,8 @@
 
 require "yast"
 
-include Yast
-include UIShortcuts
-include I18n
+include Yast::UIShortcuts
```

Comment:
I am talking only about UIShortcuts. I18n is not needed by majority of tests, so there is no pressure to add it. But UI shortcuts are useful for any UI testing.